documentation/docs/user_guide/running_cable/running_cable_with_casacnp.md

```diff
@@ -0,0 +1,57 @@
+| `cable_user%CASA_fromZero = .TRUE.`      | Enable cold start without CASA restart file.                 |
+| `casafile%cnpbiome = <input_file_path>`  | Input file path to PFT lookup table used by CASA-CNP.        |
+| `casafile% = <input_file_path>`          | Input file path to vegetation phenology file. The vegetation phenology file contains a latitudinal listing of phenology dates for various vegetation types. |
+| `casafile%cnpepool = <output_file_path>` | Output file path to the CASA restart file containing CASA state variables. It can be fed back to another simulation for the purpose of spin-up via `casafile%cnpipool`. |
+| `casafile%cnpflux = <output_file_path>`  | Output file path to CASA output fluxes in CSV (`.csv`) format. |
+| `casafile%out = <output_file_path>`      | Output file path to CASA output variables in NetCDF (`.nc`) format. |
+
+To run CABLE with CASA-CNP from a CASA restart file (warm start), set `cable_user%CASA_fromZero = .FALSE.` and `casafile%cnpipool` to the input file path of the CASA restart file.
+
+## Running CASA-CNP in offline mode:
+
+It can be useful to run CASA-CNP in offline (or CASA only) mode for the purpose of spinning up CASA-CNP more efficiently. Since CASA-CNP operates on a daily time step as opposed to CABLE which inherently runs on a sub-diurnal time step, CABLE makes up a significant fraction of CABLE-CASA compute. For the purposes of spinning up CASA-CNP, two way coupling between CABLE and CASA-CNP is not strictly necessary and CASA-CNP can be forced using CABLE to CASA input meteorology. This can reduce the amount of compute time spent on CASA-CNP spin-up.
+
+To run CASA-CNP in offline mode, CABLE to CASA dump files need to first be generated for each model year throughout spin-up period. The CABLE to CASA dump files can be generated by enabling the CABLE namelist option `cable_user%CASA_DUMP_WRITE = .TRUE.`. By default, the dump files will be written to a path prefixed by the `casafile%c2cdumppath` CABLE namelist option.
+
+Once the CABLE to CASA dump files have been generated for the spin-up period, CASA-CNP can be run in offline mode by enabling the CABLE namelist options `CASAonly = .TRUE.` and `cable_user%CASA_DUMP_READ = .TRUE.` (and `cable_user%CASA_DUMP_WRITE = .FALSE.` to prevent overwriting the CABLE to CASA input meteorology). The CABLE to CASA input file paths will be read from the same path as they were written initially. Also, note that `CASAonly` may be set implicitly by setting `icycle` to a value greater than `10`.
+
+## Accelerated spin-up (semi-analytical solution method):
+
+To avoid running long and computationally expensive spin-up simmulations, CABLE contains a semi-analytical solution method which reduces the spin-up time ([Xia et al. 2012][gmd-5-1259-2012]). Note, this method should be used only for simulations where the phosphorus cycle and its coupling with carbon and nitrogen cycles has been switched off (i.e. `icycle = 2`).
```

Comment:
```suggestion
To avoid running long and computationally expensive spin-up simulations, CABLE contains a semi-analytical solution method that reduces the spin-up time ([Xia et al. 2012][gmd-5-1259-2012]). Note, this method should be used only for simulations where the phosphorus cycle and its coupling with carbon and nitrogen cycles have been switched off (i.e. `icycle = 2`).
```